paymentrequest/payment-handler/index.html

```diff
@@ -7,11 +7,11 @@
 <h3>Background</h3>
 <p>
   <a href="https://github.com/w3c/browser-payment-api">PaymentRequest</a> lets
-  you accept payment with a URL-based payment method.
+  you accept payment from different payment methods.
 </p>
 
 <p>
-  This sample accepts the a URL-based payment method.
+  This sample accepts the the URL-based payment method.
```

Comment:
Please remove the duplicate "the"

Reply:
Ack. It has been fixed in the last commit.